<Exception>
<DateTime>9/8/2015 12:33:03 PM</DateTime>
<SessionID>biijfzpfrj001yjojnsnvtln</SessionID>
<Summary>An unhandled exception occured on Tuesday, September 8, 2015 12:33:03 PM</Summary>
<Info>
<Verb>POST</Verb>
<URL>ustxri00dbamgt/dbaweb/UpgradeServerForm.aspx</URL>
<HostIP>10.172.203.39</HostIP>
<ReffererURL>ustxri00dbamgt/dbaweb/UpgradeServerForm.aspx</ReffererURL>
</Info>
<Browser>
<Name>Chrome</Name>
<Version>45.0</Version>
<CookieSupport>True</CookieSupport>
<JSSupport>True</JSSupport>
<Platform>WinNT</Platform>
</Browser>
<User>Guest</User>
<ErrorMessage>System.Data.SqlClient.SqlException (0x80131904): Violation of PRIMARY KEY constraint 'PK_InventoryTracking'. Cannot insert duplicate key in object 'dbo.InventoryTracking'. The duplicate key value is (USTXRIHCMDBQ).
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite)
   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)
   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   at DBAWebsite.UpgradeServerForm.myBtn_Click(Object sender, EventArgs e) in h:\DBAWebsite\UpgradeServerForm.aspx.cs:line 299
ClientConnectionId:18028ce1-d4b7-4135-9039-c28209717dc6
Error Number:2627,State:1,Class:14</ErrorMessage>

</Exception>

<Exception>
<DateTime>9/8/2015 12:33:04 PM</DateTime>
<SessionID>biijfzpfrj001yjojnsnvtln</SessionID>
<Summary>An unhandled exception occured on Tuesday, September 8, 2015 12:33:04 PM</Summary>
<Info>
<Verb>POST</Verb>
<URL>ustxri00dbamgt/dbaweb/UpgradeServerForm.aspx</URL>
<HostIP>10.172.203.39</HostIP>
<ReffererURL>ustxri00dbamgt/dbaweb/UpgradeServerForm.aspx</ReffererURL>
</Info>
<Browser>
<Name>Chrome</Name>
<Version>45.0</Version>
<CookieSupport>True</CookieSupport>
<JSSupport>True</JSSupport>
<Platform>WinNT</Platform>
</Browser>
<User>Guest</User>
<ErrorMessage>System.Data.SqlClient.SqlException (0x80131904): Violation of PRIMARY KEY constraint 'PK_InventoryTracking'. Cannot insert duplicate key in object 'dbo.InventoryTracking'. The duplicate key value is (USTXRIHCMDBQ).
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite)
   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)
   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   at DBAWebsite.UpgradeServerForm.myBtn_Click(Object sender, EventArgs e) in h:\DBAWebsite\UpgradeServerForm.aspx.cs:line 299
ClientConnectionId:18028ce1-d4b7-4135-9039-c28209717dc6
Error Number:2627,State:1,Class:14</ErrorMessage>

</Exception>

<Exception>
<DateTime>9/8/2015 12:34:34 PM</DateTime>
<SessionID>biijfzpfrj001yjojnsnvtln</SessionID>
<Summary>An unhandled exception occured on Tuesday, September 8, 2015 12:34:34 PM</Summary>
<Info>
<Verb>POST</Verb>
<URL>ustxri00dbamgt/DBAWeb/DecommissionServerForm.aspx</URL>
<HostIP>10.172.203.39</HostIP>
<ReffererURL>ustxri00dbamgt/DBAWeb/DecommissionServerForm.aspx</ReffererURL>
</Info>
<Browser>
<Name>Chrome</Name>
<Version>45.0</Version>
<CookieSupport>True</CookieSupport>
<JSSupport>True</JSSupport>
<Platform>WinNT</Platform>
</Browser>
<User>Guest</User>
<ErrorMessage>System.Net.Mail.SmtpException: Failure sending mail. ---> System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.Mail.SmtpPooledStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   at System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at DBAWebsite.DecommissionServerForm.send_mail() in h:\DBAWebsite\DecommissionServerForm.aspx.cs:line 136</ErrorMessage>
<InnerException>System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.Mail.SmtpPooledStream.Dispose(Boolean disposing)
   at System.IO.Stream.Close()
   at System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   at System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)</InnerException>

</Exception>

